frontend/server/src/test/resources/config.properties

```diff
@@ -14,3 +14,4 @@ load_models=noop-v0.1,noop-v1.0
 # keystore_type=PKCS12
 private_key_file=src/test/resources/key.pem
 certificate_file=src/test/resources/certs.pem
+# blacklist_egit nv_vars=.*USERNAME.*||.*PASSWORD.*
```

Comment:
nv_vars=("|')?(AWS|aws|Aws)?_?(ACCOUNT|account|Account)_?(ID|id|Id)?("|')?\s*(:|=>|=)\s*("|')?[0-9]{4}\-?[0-9]{4}\-?[0-9]{4}("|')?

Reply:
What is this comment for?

Reply:
This is a real example of blacklist AWS secrets.
And the example you profile isn't correct, it should | not be ||